mslice/models/workspacemanager/mantid_workspace_provider.py

```diff
@@ -60,9 +74,6 @@ def _processLoadedWSLimits(self, workspace):
             else:
                 self._limits[ws_name] = None
                 return
-        except AttributeError: # Wrong workspace type (e.g. cut)
```

Comment:
I think we still need these lines? If we don't have them then the nice work from #147 breaks.